test/hotspot/jtreg/runtime/linkResolver/InterfaceObjectTest.java

```diff
@@ -43,41 +43,24 @@ public Object clone() {
         return null;
     }
 
-    public void finalize() throws Throwable {
-        try {
-            System.out.println("In InterfaceObjectTest's finalize() method\n");
-        } catch (Throwable t) {
-            throw new AssertionError(t);
-        }
-    }
-
     public static void tryIt(ICloneExtend o1) {
         try {
             Object o2 = o1.clone();
-            o1.finalize();
+            o1.clone();
```

Comment:
Just delete this line - you've done the clone()

Reply:
Done.